js/app.js

```diff
@@ -2,257 +2,282 @@
-var guesses = 0;
-var livedString = 'I have lived in: ';
-
-for (var x = 0; x < countries.length; x++) {
-  if (countries[x] === 'usa' || countries[x] === 'uk') {
-    countries[x] = countries[x].toUpperCase();
-  } else {
-    countries[x] = countries[x].charAt(0).toUpperCase() + countries[x].slice(1);
-  }
-
-  if (x === countries.length - 1) {
-    livedString += 'and ' + countries[x] + '.';
-  } else {
-    livedString += countries[x] + ', ';
-  }
-}
+function question7(){
+  var countryField = document.getElementById('country');
+  var countries = ['usa', 'russia', 'china', 'latvia', 'uk'];
+  var guesses = 0;
+  var livedString = 'I have lived in: ';
+
+  for (var x = 0; x < countries.length; x++) {
+    if (countries[x] === 'usa' || countries[x] === 'uk') {
+      countries[x] = countries[x].toUpperCase();
+    } else {
+      countries[x] = countries[x].charAt(0).toUpperCase() + countries[x].slice(1);
+    }
 
-// do/while loop asks question over and over, if necessary
-do {
-  var country = prompt('Name a country I have lived in for longer than a month.').toLowerCase();
-  var inArray = false;
-
-  // loop over countries array and checks if user information matches any contents
-  for (var j = 0; j < countries.length; j++) {
-
-    if (country === countries[j].toLowerCase()) {
-      console.log('Question 7: You are correct! ' + livedString);
-      alert('You are correct! ' + livedString);
-      inArray = true;
-      countryField.className = 'correct';
-      countryField.innerHTML = 'Your answer: ' + country +
-            '; ---> My answers: ' + livedString;
-      numberCorrect += 1;
-      guesses = 7;
-      break;
+    if (x === countries.length - 1) {
+      livedString += 'and ' + countries[x] + '.';
     } else {
-      continue;
+      livedString += countries[x] + ', ';
     }
   }
 
-  guesses += 1;
-  if (inArray === false) {
-    console.log('Question 7: You are wrong.');
-    alert('You are wrong, guess again.');
-    if (guesses === 6) {
-      countryField.innerHTML = 'Your answer: ' + country +
-            '; ---> My answers: ' + livedString;
-      countryField.className = 'incorrect';
+  // do/while loop asks question over and over, if necessary
+  do {
+    var country = prompt('Name a country I have lived in for longer than a month.').toLowerCase();
+    var inArray = false;
+
+    // loop over countries array and checks if user information matches any contents
+    for (var j = 0; j < countries.length; j++) {
+
+      if (country === countries[j].toLowerCase()) {
+        console.log('Question 7: You are correct! ' + livedString);
+        alert('You are correct! ' + livedString);
+        inArray = true;
+        countryField.className = 'correct';
+        countryField.innerHTML = 'Your answer: ' + country +
+              '; ---> My answers: ' + livedString;
+        numberCorrect += 1;
+        guesses = 7;
+        break;
+      } else {
+        continue;
```

Comment:
Just so you know, you do not need a `continue` statement here since there is no code after this in the loop.